### ISSUE TITLE
Controlled TimeInput cannot be controlled

### ISSUE THREAD

**Issue by zifeo** [mantinedev/mantine#377]
@mantine/dates@3.1.2 with latest Safari

```
import { useState } from 'react';
import { TimeInput } from '@mantine/dates';

function Demo() {
  const [value, onChange] = useState(new Date());
  useEffect(() => {
    const date = // async fetch
    onChange(date); // date does not change
  }, []);
  return <TimeInput value={value} onChange={onChange} />;
}
```

I suspect this is caused by the lack of a default/empty value in the component.

**Comment by rtivital** [comment]
I've released a patch with a fix, please update to `3.1.4` and report if everything is working correctly

**Comment by zifeo** [comment]
@rtivital That's fast, works!
